mkosi/qemu.py

```diff
@@ -808,7 +808,7 @@ def run_qemu(args: Args, config: Config) -> None:
     cmdline: list[PathString] = [
         find_qemu_binary(config),
         "-machine", machine,
-        "-smp", str(config.qemu_smp),
+        "-smp", str(config.qemu_smp if config.qemu_smp != 0 else os.cpu_count()),
```

Comment:
One last nit. This could be
```suggestion
        "-smp", str(config.qemu_smp or os.cpu_count()),
```